crates/swc_css_codegen/src/lib.rs

```diff
@@ -764,7 +764,11 @@ where
 
     #[emitter]
     fn emit_str(&mut self, n: &Str) -> Result {
-        self.wr.write_raw(Some(n.span), &n.raw)?;
+        if self.config.minify {
+            self.wr.write_str(Some(n.span), &n.value)?;
+        } else {
+            self.wr.write_raw(Some(n.span), &n.raw)?;
+        }
```

Comment:
That is why we need `raw` and `value`, also `raw` can be used by plugins and in other tools without re parsing original source, because parsing some things in CSS is not easy and require to implement lexer

Reply:
Oh, I see. Great to see actual usecases of these fields.